### PR TITLE
cleanup: Missing requires

### PR DESCRIPTION
**Pull request by maage**
Several interfaces were missing requires.

### CODE REVIEW

File: policy/modules/system/systemd.if [hunk 1]
```diff
@@ -1417,6 +1417,7 @@ interface(`systemd_status_networkd',`
 interface(`systemd_relabelfrom_networkd_tun_sockets',`
 	gen_require(`
 		type systemd_networkd_t;
+		class tun_socket relabelfrom;
```
Comment (pebenito) [inline policy/modules/system/systemd.if:1420]: This is not necessary.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/system/systemd.if (hunk 1 above)
Comment (pebenito) [reply policy/modules/system/systemd.if:1420]: To further clarify, `tun_socket` is a kernel object class, so it is always requried.

Comment (maage) [reply policy/modules/system/systemd.if]: Yes, I made some mistake here. Now when I checked these, I was not able to trigger the issue.

**Comment by maage** [comment]
Dropper unnecessary patch as required.

**Review by pebenito — APPROVED** [review]

**Pull request merged**